playground/Zametaev/server/app.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask
+from flask_pymongo import PyMongo
+
+app = Flask(__name__)
+
+MONGO_HOST = "mongo"
+MONGO_PORT = 27017
+MONGO_DB = "myDatabase"
+
+app.config["MONGO_URI"] = f"mongodb://{MONGO_HOST}:{MONGO_PORT}/"
```

Comment:
```suggestion
app.config["MONGO_URI"] = f"mongodb://{MONGO_HOST}:{MONGO_PORT}/{MONGO_DB}"
```
Потерял тут название БД, к которой обращаться, поэтому на стороне сервера срабатывало исключение.